stack.yaml

```diff
@@ -1,5 +1,13 @@
----
-resolver: lts-10.0
+resolver: lts-10.6
+
 packages:
-  - '.'
-extra-deps: []
+- '.'
+
+extra-deps:
+  - conduit-1.3.0
+  - conduit-extra-1.3.0
+  - persistent-2.8.1
+  - resourcet-1.2.0
+  - yesod-core-1.6.1
+  - yesod-form-1.6.1
+  - yesod-persistent-1.6.0
```

Comment:
Are these strictly needed? I have no issue with bumping upper bounds, but I'd like to avoid extra-deps here since it affects CI and could give false-negatives (positives?) when checking the nightly build, etc.

Reply:
I've discussed this with a co-maintainer before, I'll remove it for now but share the reasoning:

If I bump dependency bounds to support some new version of a library or libraries, it makes sense to me that I'd want to pin those newest versions and check that the project continues to build against them. Arguably it's slightly more necessary here as much of this isn't in a stable LTS yet.

Alternative would be to have multiple `stack.yaml`s beyond the usual assortment for different GHC versions, one tracking nightly + pinned versions.

Also it is sort of a "show your work" thing. Shows I was actually building the library against the versions the bounds bump was permitting. I can see why you might not want it, so I'll remove it.

Reply:
Thanks for backing it out. That reasoning makes sense and I will give it some more thought. I've been moving towards more comprehensive CI in #44, which includes a build on nightly. So maybe we can do something once that lands that exercises these newer versions explicitly.

I _think_ my stance is that a CI of supported versions + nightly is enough. In other words, I'm comfortable understanding that nightly may or may not _currently_ catch the newest versions our bounds allow, but that when it matters it will. Does that make sense?

Reply:
Makes sense to me. :+1: